### PR TITLE
Feature/refactor for tracking progress

### PR DESCRIPTION
**Pull request by gtvj**
Hi @AshTNA - could you look at this PR please. It's a significant overhaul of the repository to support the traceability of how the Explorer app will function (intended to mirror what we did in Kew Booking System). 

My intention here is to provide, for any given prototype, a clear sense of how and why it differs from its predecessor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,135 +1,77 @@
...
+This [version](iterations/code/iteration_one) was the first produced within the Beta timebox for project Etna. It represented a significant revision of the [time explorer](/alpha_time_explorer.md) which was produced during the Alpha phase with many of the changes reflecting necessary accessibility and usability improvements. 
 
-[Historpedia.js](https://js.histropedia.com/index.html) has been mentioned as an approach that might be worth considering. This is a JavaScript-powered visualisation tool.
+This version was user tested with six participants in December 2020 and recommendations from the [user testing report](user_research/iteration_one_user_testing_report.pdf) were taken forward into [iteration two](#iteration-two).
 
-# Proposals
+## User testing recommendations
 
-The _current_ proposals the next iteration included:
+### Testing of iteration two
```
Comment (AshGDS) [inline README.md]: Hi Gwyn, is this heading needed still?

**Review by AshGDS — CHANGES_REQUESTED** [review]
Hi Gwyn, everything looks good, there's just this one small change to consider. Thanks again :+1:

File: README.md (hunk 1 above)
Comment (gtvj) [reply README.md]: All sorted. Thanks for spotting this 👍

**Review by AshGDS — APPROVED** [review]
All good :+1:

**Pull request merged**